src/startupscriptgenerator/src/dotnetcore/scriptgenerator.go

```diff
@@ -65,7 +60,7 @@ func (gen *DotnetCoreStartupScriptGenerator) GenerateEntrypointScript(scriptBuil
 			scriptBuilder.WriteString("fi\n")
 		}
 	} else {
-		if builtByOryx {
+		if gen.Manifest.Exists {
```

Comment:
consider my comment above: this could have been `if gen.Manifest != nil {`

Reply:
see my comment above regarding using `nil` check